### ISSUE TITLE
rebroadcasting LN force-close tx: Transition not allowed: FORCE_CLOSING -> FORCE_CLOSING

### ISSUE THREAD

**Issue by SomberNight** [spesmilo/electrum#5824]
```
Traceback (most recent call last):
  File "...\electrum\electrum\util.py", line 999, in wrapper
    return await func(*args, **kwargs)
  File "...\electrum\electrum\lnworker.py", line 695, in on_update_open_channel
    await self.force_close_channel(chan.channel_id)
  File "...\electrum\electrum\lnworker.py", line 1253, in force_close_channel
    chan.set_state(channel_states.FORCE_CLOSING)
  File "...\electrum\electrum\lnchannel.py", line 225, in set_state
    raise Exception(f"Transition not allowed: {old_state.name} -> {state.name}")
Exception: Transition not allowed: FORCE_CLOSING -> FORCE_CLOSING
```

Maybe we should allow no-op state transitions?
Although that might silence some bugs potentially; not sure.

**Comment by ecdsa** [comment]
we can explicitly allow `FORCE_CLOSING -> FORCE_CLOSING`

**Comment by ecdsa** [comment]
fixed in 36f32651cc970b3429b56aa1a763374c50eee980
